lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -168,6 +177,18 @@ private void buildCompositionLayer() {
         this, Layer.Factory.newInstance(composition), composition.getLayers(), composition);
   }
 
+  private void applyColorFilters() {
+    if (compositionLayer == null) {
+      return;
+    }
+
+    ColorFilterData[] data = (ColorFilterData[]) colorFilterData.toArray();
```

Comment:
i should clarify that the code you presented is definitely possible, but there is not a method to retrieve the context with just the index without the array